onnx_tf/backend.py

```diff
@@ -208,6 +206,10 @@ class TensorflowBackend(Backend):
   }
 
   @classmethod
+  def pad2tfpad(cls, pads):
+    return "VALID" if pads[0] == 0 else "SAME"
```

Comment:
I didn't understand the ONNX pads parameter from the documentation.  Why does it have multiple dimensions?

Reply:
it has 2 times the spatial dims number of elements. 2 for each spacial axis indicating pads in the beginning/end of this dimension.

Reply:
Does tensorflow support adding such fine-grained control for padding?  It doesn't seem like it, from the documentation.

Reply:
No it does not, this pad2tfpad follows a simple logic: if the first element is 0 then return "VALID" which is usually no padding and if not return "SAME" which has padding. So far it works. We will eventually remove it in favor of the auto_pad attribute in the latest PR.